services/ec2/src/main/java/org/finra/gatekeeper/controllers/AccessRequestController.java

```diff
@@ -16,6 +16,7 @@
 
```

Comment:
Now that i'm thinkng about it we should expose these changes in the common package since the EC2 and RDS are using these values exactly the same, this removes the need to duplicate the code across both of the backend services.